cognite/client/_api/sequences.py

```diff
@@ -0,0 +1,403 @@
+import math
+from typing import *
+
+from cognite.client import utils
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes import Sequence, SequenceFilter, SequenceList, SequenceUpdate
+from cognite.client.utils._experimental_warning import experimental_api
+
+
+@experimental_api(api_name="Sequences")
+class SequencesAPI(APIClient):
+    _RESOURCE_PATH = "/sequences"
+    _LIST_CLASS = SequenceList
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.data = SequencesDataAPI(*args, **kwargs)
+
+    def __call__(
+        self, chunk_size: int = None, asset_ids: List[int] = None, limit: int = None
```

Comment:
According to @MariuszCognite this should take an `externalId` parameter, but there is no `assetIds` filtering.